src/resolvers/answer.ts

```diff
@@ -144,6 +152,115 @@ export const resolvers: Resolvers = {
+
+        context.logger.error(prepareObjectForLogs(err), `Failure in ${reference}`);
+        throw InternalServerError();
+      }
+    },
+    // Update answer comment
+    updateAnswerComment: async (_, { answerCommentId, answerId, commentText }, context: MyContext): Promise<AnswerComment> => {
+      const reference = 'updateAnswerComment resolver';
+      try {
+        if (isAuthorized(context.token)) {
+          const answer = await Answer.findById(reference, context, answerId);
+
+          if (!answer) {
+            throw NotFoundError(`Answer with ID ${answerId} not found`);
+          }
+
+          const plan = await Plan.findById(reference, context, answer.planId);
+          if (!plan) {
+            throw NotFoundError(`Plan ${answer.planId} not found`);
+          }
+          const project = await Project.findById(reference, context, plan.projectId);
+
+          if (await hasPermissionOnProject(context, project)) {
+            const answerComment = await AnswerComment.findById(reference, context, answerCommentId);
+
+            if (!answerComment) {
+              throw NotFoundError(`Answer comment ${answerCommentId} not found`);
+            }
+
+            // Only user who added the comment can update it
+            if (answerComment.createdById === context.token.id) {
+              answerComment.commentText = commentText;
+              return await answerComment.update(context);
+            }
+
+            throw ForbiddenError(`Inadequate permission to update answer comment ${answerComment.id}`)
+
+          }
+        }
+        throw context?.token ? ForbiddenError() : AuthenticationError();
+      } catch (err) {
+        if (err instanceof GraphQLError) throw err;
+
+        context.logger.error(prepareObjectForLogs(err), `Failure in ${reference}`);
+        throw InternalServerError();
+      }
+    },
+    // Remove answer comment
+    removeAnswerComment: async (_, { answerCommentId, answerId }, context: MyContext): Promise<AnswerComment> => {
+      const reference = 'removeAnswerComment resolver';
+      try {
+        if (isAuthorized(context.token)) {
+          const answer = await Answer.findById(reference, context, answerId);
+
+          if (!answer) {
+            throw NotFoundError(`Answer with ID ${answerId} not found`);
+          }
+
+          const plan = await Plan.findById(reference, context, answer.planId);
+          if (!plan) {
+            throw NotFoundError(`Plan ${answer.planId} not found`);
+          }
+          const project = await Project.findById(reference, context, plan.projectId);
+
+          if (await hasPermissionOnProject(context, project)) {
+            const answerComment = await AnswerComment.findById(reference, context, answerCommentId);
+
+            if (!answerComment) {
+              throw NotFoundError(`Answer comment ${answerCommentId} not found`);
+            }
+
+            return await answerComment.delete(context);
```

Comment:
we probably only want the user who created the comment to delete it. so use `if (answerComment.createdById === context.token.id) {` here as well